en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md

```diff
@@ -315,6 +315,33 @@ docker build -t <IMAGE_NAME> -f <DOCKER_FILE_PATH> <CONTEXT>
 docker build -t myimages/choreo-connect-router:{{choreo_connect.version}} -f Dockerfile .
 ```
 
+### Mount Dropins Directory to a Choreo Connect Helm Deployment
```

Comment:
I think we do not want to mention about helm, because this is prod guide page
```suggestion
### Mount Files into Dropins Directory (Optionally)
```